depmapomics/qc/config.py

```diff
@@ -59,11 +65,16 @@
 
 LINES_TO_DROP_COMMON = {"ACH-001108", "ACH-001011", "ACH-001092"}
 LINES_TO_DROP_DNA = LINES_TO_DROP_COMMON
-LINES_TO_DROP_RNA = LINES_TO_DROP_COMMON
+LINES_TO_DROP_RNA = LINES_TO_DROP_COMMON | {"ACH-002778", "ACH-002745"}
```

Comment:
why do we drop these lines?

Reply:
let's add a comment

Reply:
need to check if qc looks at the blacklist column

Reply:
@5im1z